src/redux/reducer.ts

```diff
@@ -1738,24 +1738,28 @@ function updateEquipments(currentEquipments: Identifiable[], newOrUpdatedEquipme
     return currentEquipments;
 }
 
-function synchCurrentTreeNode(state: AppState, nextCurrentNodeUuid: string) {
+function synchCurrentTreeNode(state: AppState, nextCurrentNodeUuid: UUID | undefined) {
     const nextCurrentNode = state.networkModificationTreeModel?.treeNodes.find(
         (node) => node?.id === nextCurrentNodeUuid
     );
-    //  we need to overwrite state.currentTreeNode to consider label change for example.
-    state.currentTreeNode = { ...nextCurrentNode };
+    if (nextCurrentNode) {
+        //  we need to overwrite state.currentTreeNode to consider label change for example.
+        state.currentTreeNode = { ...nextCurrentNode };
+    }
 }
 
-function unravelSubTree(treeModel: NetworkModificationTreeModel, subtreeParentId: string, node: Node<TreeNodeData>) {
+function unravelSubTree(
+    treeModel: NetworkModificationTreeModel,
+    subtreeParentId: string,
+    node: NetworkModificationNode | RootNode | null
```

Comment:
I'm confused, unravelSubTree is called with a list as a 3rd parameter, how is it supposed to work ?